build

```diff
@@ -28,6 +28,20 @@ elif [ "$(uname -s)" = 'Linux' ]; then
     SYSTEM="$(echo "${DIST}${VERSION_ID}" | sed -r 's/([a-zA-Z]*)-.*/\1/g')"
 fi
 
+ARCH="$(uname -m)"
+case "$ARCH" in
+    x86_64)
+        ARCH='amd64'
+        ;;
+    aarch64)
+        ARCH='arm64'
+        ;;
+    arm*)
+        ARCH=arm
```

Comment:
@zmstone  I want to rename `armv6`, `armv7`, `armhf` to `arm`, what do you think?

Reply:
if `SYSTEM` can tell them apart, it's fine.